meta-resin-common/recipes-support/resin-mounts/resin-mounts/var-lib-systemd.mount

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Systemd presistent state
+Requires=mnt-conf.mount
+After=mnt-conf.mount
```

Comment:
I think we need some changes here. @telphan can you please sync with @michal-mazurek and sync on the implementation details of the readme architecture? As well, I would like this documented somewhere. I would say in README for now.